### DISCUSSION TITLE
Stylesheets not working properly

### DISCUSSION THREAD

**Discussion by h4rvey-g** [ObsidianPublisher/obsidian-github-publisher#175] [Q&A — Template] [answered]
I use github actions to deploy the website, and it runs successfully. But the website appears without stylesheets. How should I fix it? Here is my site: [site](https://www.guolab.live)

**Comment by Mara-Li** [comment]
Without the repo or the mkdocs.yam I can't help you.

**Comment by h4rvey-g** [comment]
My bad.
repo: [babaolanqiu/Notes-blog](https://github.com/babaolanqiu/Notes-blog)

**Comment by Mara-Li** [comment]
Did you try locally?

**Comment by h4rvey-g** [comment]
yes, it can run normally locally.

**Reply by Mara-Li** [reply]
To be honest, I don't see anything during the build to know why there is a problem.
Seems to be more [Mkdocs Material](https://github.com/squidfunk/mkdocs-material/discussions) error.

**Reply by Mara-Li** [reply, marked as answer]
EDIT:
The link to your blog is https://www.guolab.live/
it's not the link in the mkdocs.yaml site_url

**Reply by h4rvey-g** [reply]
Now it's working! Thanks!
